cmd/init.go

```diff
@@ -126,6 +140,10 @@ dapr init --from-dir <path-to-directory>
 	},
 }
 
+func warnForPrivateRegFeat() {
+	print.WarningStatusEvent(os.Stdout, "Flag --image-registry is a preview feature and is subject to change. It is only available from CLI version 1.7 onwards.")
```

Comment:
```suggestion
	print.WarningStatusEvent(os.Stdout, "Flag --image-registry is a preview feature and is subject to change.")
```

Reply:
since the flag will be partially available in 1.7 and now additionally available with two different commands in 1.8